ui/app/sample-data/demoDashboard.ts

```diff
@@ -159,47 +159,19 @@ const demoDashboard: DashboardResource = {
                       kind: 'PrometheusTimeSeriesQuery',
                       spec: {
                         query:
-                          'node_memory_MemTotal_bytes{job="node",instance=~"$instance"} - node_memory_MemFree_bytes{job="node",instance=~"$instance"} - node_memory_Buffers_bytes{job="node",instance=~"$instance"} - node_memory_Cached_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_Buffers_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_Cached_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_MemFree_bytes{job="node",instance=~"$instance"}',
+                          '1 - node_filesystem_free_bytes{job="node",instance=~"$instance",fstype!="rootfs",mountpoint!~"/(run|var).*",mountpoint!=""} / node_filesystem_size_bytes{job="node",instance=~"$instance"}',
+                        legendFormat: '{{instance}} {{job}}', // figure this syntax out with Shan / Luke
                       },
                     },
                   },
                 },
               ],
-              show_legend: true,
-              unit: { kind: 'Bytes' },
+              legend: {
+                show: true,
+                type: 'compact', // optional, eventually we'll have 'table' in addtition to default 'compact'
+                position: 'bottom', // optional, defaults to bottom
+              },
+              unit: { kind: 'Percent' },
```

Comment:
This looks great to me 👍

Reply:
Another thought, we'll need to figure out how we're going to do multiple y axes in the future (unit will need to move at that point). We could do a y_axes array, but I think it'll depend on if we want a max of 2 (primary on left, secondary on right) or will users need to be able to add more than 2?

Reply:
Oh that is interesting...

Would be maximum flexibility to plan for more than 2 axes...

But my knee-jerk reaction says 2 axes covers the majority of use cases, and it's enough to plan for that / planning for more might be extra abstraction that doesn't help us much. 

But maybe there's some data could help us know if that's true